js/superHero.js

```diff
@@ -0,0 +1,99 @@
+class superHeroMovies {
+
+	isSequel = true || false;
+    isStreamed = true || false;
+
+	constructor(props) {
+		this.isSequel = props.isSequel;
```

Comment:
If these are always passed as props, no need to declare as public props of the class. You would reserve those for locally global variables that you want to use to store some changeable state in memory for each instance.

Reply:
Edited it. I went out of my way to read the following below as well.

https://blog.sessionstack.com/how-javascript-works-memory-management-how-to-handle-4-common-memory-leaks-3f28b94cfbec